pkg/api/metadata.go

```diff
@@ -69,14 +85,20 @@ func (m *Metadata) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // ServeHTTP sends API version v1 documentation response.
 func (v *VersionMetadata) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	version, ok := GetCommitSha()
+	if ok != nil {
+		version = ok.Error()
+	}
```

Comment:
Usually, `ok` is a boolean. Here it is of type `error`, so I believe we should rename it to `err`.